pkg/sql/storageparam/tablestorageparam/table_storage_param.go

```diff
@@ -227,13 +237,41 @@ var tableParams = map[string]tableParam{
 			return nil
 		},
 		onReset: func(po *Setter, evalCtx *eval.Context, key string) error {
-			return errors.WithHintf(
-				pgerror.Newf(
+			if po.tableDesc.RowLevelTTL != nil {
+				po.tableDesc.RowLevelTTL.DurationExpr = ""
+			}
+			return nil
+		},
+	},
+	`ttl_expiration_expression`: {
+		onSet: func(ctx context.Context, po *Setter, semaCtx *tree.SemaContext, evalCtx *eval.Context, key string, datum tree.Datum) error {
+			stringVal, err := paramparse.DatumAsString(evalCtx, key, datum)
+			if err != nil {
+				return err
+			}
+			stringVal = strings.TrimSpace(stringVal)
+			if !strings.HasPrefix(stringVal, "(") && !strings.HasSuffix(stringVal, ")") {
+				stringVal = "(" + stringVal + ")"
+			}
+			// todo(wall): add type checking
```

Comment:
can you make this an issue number to follow? you can re-use the same issue number if you're not closing it